doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -0,0 +1,538 @@
+$primary-color: #0288D1;
+$secondary-color: #BDBDBD;
+
+$custom-theme-palette: palette($primary: $primary-color, $secondary: $secondary-color);
+```
+
+And then, with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color), we can easily retrieve the color from the palette. 
+
+```scss
+$custom-grid-theme: grid-theme(
+    $palette: $custom-theme-palette,
+    $resize-line-color: color($custom-theme-palette, 'secondary', 500)
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please, refer to [`Palettes`](../themes/sass/palettes.md) topic for detailed guidance on how to use them.
+
+### Using Schemas
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/sass/schemas.md). A **schema** is a recipe of a theme.
+
+Extend the predefined schema provided for every component, in this case - [`light-grid`]({environment:sassApiUrl}/index.html#variable-_light-grid) schema:
+
+```scss
+// Extending the light grid schema
+$light-grid-schema: extend($_light-grid,
+    (
+        resize-line-color: (
+           color: ('secondary', 500)
+            ),
+        header-background: (
+           color: ("primary", 100)
+            ),
+        header-text-color: (
+           color: ("primary", 600)
+            )
+    )
+);
+```
+
+In order to apply our custom schema, we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component theme:
+
+```scss
+// Extending the global light-schema
+$custom-light-grid-schema: extend($light-schema,(
+    igx-grid: $light-grid-schema
+));
+
+// Specifying the palette and schema of the custom grid theme
+$custom-grid-theme: grid-theme(
+    $palette: $custom-theme-palette,
+    $schema: $custom-light-grid-schema
+);
+```
+Don't forget to include the theme in the same way as it was demonstrated above.
+
+### Demo
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-resize-line-styling-sample" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-resize-line-styling" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}{ComponentSample}-resize-line-styling" >
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
remove 2 out of 3 code-view blocks and Component build flags because will use the same routing path for Grid, TreeGrid, HierarchicalGrid in samples.